### PR TITLE
Updates and export some more utils.

### PR DESCRIPTION
**Pull request by danielpickett**
updating some types and changing some parameter names after feedback from those who know more than I do about SSOs. Also directly exporting updateTopApps.

### CODE REVIEW

File: packages/spaces/src/linkHandlers.js [hunk 1]
```diff
@@ -3,45 +3,38 @@ import nativeForm from '@availity/native-form';
...
 
   const target = getTarget(space.link.target);
   const url = !isAbsoluteUrl(space.link.url)
     ? getUrl(
-        updateUrl(
-          space.link.url,
-          'spaceId',
-          space.parents && space.parents[0] ? space.parents[0].id : linkAttributes.spaceId
-        ),
+        updateUrl(space.link.url, 'spaceId', space.parents && space.parents[0] ? space.parents[0].id : payerSpaceId),
```
Comment (jordan-a-young) [inline packages/spaces/src/linkHandlers.js]: I think we can change the last arg to be a little shorter with
```js
space.parents?.[0]?.id || payerSpaceId
```
what do you think?

Comment (danielpickett) [reply packages/spaces/src/linkHandlers.js]: You so fancy 😍 . I'll update it.

File: packages/spaces/src/linkHandlers.js [hunk 2]
```diff
@@ -3,45 +3,38 @@ import nativeForm from '@availity/native-form';
...
       )
     : space.link.url;
+
   window.open(url, target);
 };
 
-export const openSsoLink = async (space, { akaname, clientId, linkAttributes, event }) => {
+export const openLinkWithSso = async (space, { akaname, clientId, payerSpaceId, ssoParams }) => {
   if (space.metadata && space.metadata.ssoId) {
-    event.preventDefault();
```
Comment (jordan-a-young) [inline packages/spaces/src/linkHandlers.js:30]: Will this be a breaking change if someone uses it on an anchor tag?

Comment (jordan-a-young) [reply packages/spaces/src/linkHandlers.js:30]: nevermind, just saw where you are using `event.preventDefault` later

Comment (danielpickett) [reply packages/spaces/src/linkHandlers.js:30]: Thanks for bringing this up. I went back to the diff of [my original change on Sept 19](https://github.com/Availity/availity-react/commit/f061e40039a1079eafad8ce08772c206d563f5e0) and I think I'm ok. Originally the onKeyPress would check for charCode 13, then check for metadata.ssoId (again) and then prevent default. Which is what the new code is doing in the same order if I'm reading it right.

**Review by jordan-a-young — APPROVED** [review]

**Review by gregmartDOTin — APPROVED** [review]

**Pull request merged**